src/LibraryUtilities.ts

```diff
@@ -248,3 +254,39 @@ export function buildLibraryItemsFromLayoutSpecs(loadedTypes: any, layoutSpecs:
     }
     return convertToLibraryTree(typeListNodes, layoutElements);
 }
+
+function showItemRecursive(item: ItemData) {
+    item.visible = true;
+    item.childItems.forEach((childItem) => showItemRecursive(childItem));
+}
+
+export function search(text: string, item: ItemData) {
+    if (item.itemType !== "group") {
+        let index: number;
```

Comment:
Can we explicitly give `index` a value of `-1` here instead of leaving it unassigned? Because its value will be unknown if `item.searchStrings` loop below does not execute due to empty list.